core/controllers/collection_viewer.py

```diff
@@ -15,6 +15,7 @@
 """Controllers for the Oppia collection learner view."""
 
 from core.controllers import base
+from core.domain import action_decorators
```

Comment:
Let's call it core.domain.acl_decorators. I think acl ("access control") makes more sense than "action", and ACL is a standard abbreviation.